core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/boxPlot/BoxPlotOpDesc.scala

```diff
@@ -0,0 +1,137 @@
+package edu.uci.ics.texera.workflow.operators.visualization.boxPlot
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import com.kjetland.jackson.jsonSchema.annotations.{JsonSchemaInject, JsonSchemaTitle}
+import edu.uci.ics.texera.workflow.common.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.texera.workflow.common.metadata.{
+  InputPort,
+  OperatorGroupConstants,
+  OperatorInfo,
+  OutputPort
+}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{
+  Attribute,
+  AttributeType,
+  OperatorSchemaInfo,
+  Schema
+}
+import edu.uci.ics.texera.workflow.operators.visualization.{
+  VisualizationConstants,
+  VisualizationOperator
+}
+
+@JsonSchemaInject(json = """
+{
+  "attributeTypeRules": {
+    "value": {
+      "enum": ["integer", "long", "double"]
+    }
+  }
+}
+""")
+class BoxPlotOpDesc extends VisualizationOperator with PythonOperatorDescriptor {
+
+  @JsonProperty(defaultValue = "Box Plot Visual")
+  @JsonSchemaTitle("Title")
+  @JsonPropertyDescription("Add a title to your visualization")
+  var title: String = ""
+
+  @JsonProperty(value = "value", required = true)
+  @JsonSchemaTitle("Value Column")
+  @JsonPropertyDescription("Data Column for Boxplot")
+  @AutofillAttributeName
+  var value: String = ""
+
+  @JsonProperty(defaultValue = "false")
+  @JsonSchemaTitle("Horizontal Orientation")
+  @JsonPropertyDescription("Orientation Style")
+  var orientation: Boolean = _
+
+  @JsonProperty(
+    value = "Quartile Method",
```

Comment:
Is this value needed?

Reply:
Yes this the value for the field name "Quartile Method"

Reply:
Yes I think it is needed. Since in dumble plot, gantt chart, hierarchy chart, pie chart and some other charts it is also used.